examples/pme/basic_pme.py

```
@@ -0,0 +1,81 @@
+    forces = -calc_grads(energy, positions)
+
+    print(f"System: {n_atoms} atoms in {l_box}×{l_box}×{l_box} Å³ box")
+    print(f"Cutoff: {rcut} Å, Ewald threshold: {ethresh}")
+    print(f"Total PME energy: {energy.item():.6f} eV")
+    print(f"Forces shape: {forces.shape}")
+    print(
+        f"Max force magnitude: {torch.max(torch.norm(forces, dim=1)).item():.6f} eV/Å"
+    )
+
+    # Print some additional information
+    print(f"\nAdditional information:")
```

Ruff (0.14.14)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)